editor/css/dark.css

```diff
@@ -203,7 +203,6 @@ select {
 #toolbar {
 	position: absolute;
 	left: calc(50% - 290px); /* ( ( 100% - 300px ) / 2.0 ) - 140px */
```

Comment:
Removing the `width` definition is critical since the `left` computation is now not valid anymore. It seems you need a different solution if you want to correctly center a toolbar with dynamic width.

Reply:
I remove `width` to fix #16248.

Reply:
But even if this PR solves one style issue, it introduces another one.